package.json

```diff
@@ -60,7 +60,7 @@
     "ts-jest": "^28.0.3",
     "tsd": "^0.31.2",
     "typedoc": "^0.22.16",
-    "typescript": "~4.7",
```

Comment:
**remark**

We should set this to the lowest version we want to support in both `supabase-js` and here ?

Or maybe there is another way to run our types tests across multiples major versions of typescript so we can check if there is a breaking change in one of them ? 🤔 

I'm open to suggestions here.

So we reduce the chances of introducing issues in user codebases with latest TS syntax.

Reply:
I think pinning it to the oldest version we support is sufficient - TS makes breaking changes all the time but afaik we haven't bumped into an issue caused by that

Reply:
So we pin it down to `4.5.5` like I did so it matches what we have on `supabase-js` side ? Sounds good ?
Otherwise, what's the lowest version we wanna support ?

Reply:
Yep, `4.5.5.` should be enough